creditcoin-js/src/utils.ts

```diff
@@ -53,3 +53,10 @@ export const utils = (api: ApiPromise) => {
         createOverrideWeight: () => createOverrideWeight(api),
     };
 };
+
+// block time in seconds
+export const BLOCK_TIME = 15;
+
+// We currently guarantee that a given collect coins will be dealt with in 60 blocks
+// After that the task expires and you'd have to re-submit the transaction.
+export const OCW_BLOCK_DEADLINE = 60;
```

Comment:
I'm not sure where this is configured. Maybe we can expose it via the API if it isn't already exposed. 

@nathanwhit ^^^ do you have more context here ? Where does the value of 60 come from?

Reply:
> We currently guarantee that a given collect coins will be dealt with in 60 blocks (after that the task expires and you'd have to re-submit the transaction). So in the worst case `60 * 15s = 900_000`

^ That is from Nathan in #1318

Reply:
That's dependent on the value `15` which depends on the compiler flags.